## Iris Dataset Comments
***

In [32]:
# Pandas is used for storing, manipulating, and working with structured data. It allows us to investigate CSV files, amongst other features.
import pandas as pd

In [33]:
# To create plots for specific data such as bar charts, histograms.
import matplotlib.pyplot as plt

In [34]:
# Download and load the Iris data set by importing the csv file using pd.read.csv("filepath.csv")
iris = pd.read_csv('/Users/jennyformentera/Desktop/pands-project/iris.csv')


These sources were used to gain a clearer understanding of the project's requirements and analyzing what time of codes to use for summarizing the Iris dataset.

- [Python - IRIS Data visualization and explanation](https://www.kaggle.com/code/abhishekkrg/python-iris-data-visualization-and-explanation)
- [About Fisher’s Iris dataset](https://www.angela1c.com/projects/iris_project/the-iris-dataset/)
- [Iris Dataset - Exploratory Data Analysis](https://www.kaggle.com/code/lalitharajesh/iris-dataset-exploratory-data-analysis)
- [Iris Data Visualization using Python](https://www.kaggle.com/code/aschakra/iris-data-visualization-using-python)


In [35]:
iris

sepal.length  sepal.width  petal.length  petal.width    variety
0             5.1          3.5           1.4          0.2     Setosa
1             4.9          3.0           1.4          0.2     Setosa
2             4.7          3.2           1.3          0.2     Setosa
3             4.6          3.1           1.5          0.2     Setosa
4             5.0          3.6           1.4          0.2     Setosa
..            ...          ...           ...          ...        ...
145           6.7          3.0           5.2          2.3  Virginica
146           6.3          2.5           5.0          1.9  Virginica
147           6.5          3.0           5.2          2.0  Virginica
148           6.2          3.4           5.4          2.3  Virginica
149           5.9          3.0           5.1          1.8  Virginica

[150 rows x 5 columns]

In [36]:
# Look at the first row. The first row consist the headings of the dataset.
iris.iloc [0]

sepal.length       5.1
sepal.width        3.5
petal.length       1.4
petal.width        0.2
variety         Setosa
Name: 0, dtype: object

In [37]:
# Find out the unique classification/type.
iris['variety'].unique()

array(['Setosa', 'Versicolor', 'Virginica'], dtype=object)

In [38]:
# Count the number.
iris['variety'].value_counts()

variety
Setosa        50
Versicolor    50
Virginica     50
Name: count, dtype: int64

The result above provides us that there are 50 counts in each variety.

In [39]:
# Describe this data set.
iris.describe()

sepal.length  sepal.width  petal.length  petal.width
count    150.000000   150.000000    150.000000   150.000000
mean       5.843333     3.057333      3.758000     1.199333
std        0.828066     0.435866      1.765298     0.762238
min        4.300000     2.000000      1.000000     0.100000
25%        5.100000     2.800000      1.600000     0.300000
50%        5.800000     3.000000      4.350000     1.300000
75%        6.400000     3.300000      5.100000     1.800000
max        7.900000     4.400000      6.900000     2.500000

- The count specify that all the four features have 150 rows.
- As a result, from Mean we can say that Sepal is larger than Petal.

In [40]:
iris['sepal.length'].describe()

count    150.000000
mean       5.843333
std        0.828066
min        4.300000
25%        5.100000
50%        5.800000
75%        6.400000
max        7.900000
Name: sepal.length, dtype: float64